src/constants/MarkdownFiles/posts/2025-07-07-dmp-25-therealharshit-week05.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "DMP ’25 Week 05 Update by Harshit Verma"
+excerpt: "This week, I built a custom Markdown parser for VTE, began evaluating model performance, and discussed age appropriate debugging practices with mentors."
+category: "DEVELOPER NEWS"
+date: "2025-07-07"
+slug: "2025-07-07-dmp-25-therealharshit-week05"
+author: "@/constants/MarkdownFiles/authors/harshit-verma.md"
+tags: "dmp25,sugarlabs,week05,therealharshit"
+image: "assets/Images/c4gt_DMP.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 05 Progress Report by Harshit Verma
+
+**Project:** [LLM-powered Debugger for Pippy](https://github.com/sugarlabs/Pippy/issues/95)  
+**Mentors:** [Walter Bender](https://github.com/walterbender), [Ibiam Chihurumnaya](https://github.com/chimosky), [Kshitij Shah](https://github.com/kshitijdshah99)  
+**Assisting Mentors:** [Devin Ulibarri](https://github.com/pikurasa)  
+**Reporting Period:** 2025-06-30 - 2025-07-06   
+
+---
+
+## Goals for This Week
+
+- **Goal 1:** Develop a custom Markdown parser for VTE.
+- **Goal 2:** Begin model selection and optimization.
+- **Goal 3:** Discuss best debugging practices for children.
+
+---
+
+## This Week’s Achievements
+
+1. **Built a Lightweight Markdown Parser for VTE**  
```

Comment:
Please spell out what "VTE" stands for at its first instance.

Not everyone can be expected to know such jargon.